packages/core/src/features/genomic/genesFrequencyChartSlice.ts

```diff
@@ -75,12 +75,29 @@ export const fetchGeneFrequencies = createAsyncThunk<
     cohortFilters,
   }: GenomicTableProps): Promise<GraphQLApiResponse> => {
     const caseFilters = buildCohortGqlOperator(cohortFilters);
+    const cohortFiltersContent = caseFilters?.content
+      ? Object(caseFilters?.content)
+      : [];
 
     const graphQlVariables = {
-      caseFilters: caseFilters,
       geneFrequencyChart_filters: buildCohortGqlOperator(genomicFilters) ?? {},
       geneFrequencyChart_size: pageSize,
       geneFrequencyChart_offset: offset,
+      geneCaseFilter: {
+        content: [
+          ...[
```

Comment:
This could just be 
`
{
              content: {
                field: "cases.available_variation_data",
                value: ["ssm"],
              },
              op: "in",
            }
`

instead of 
`
...[
     {
              content: {
                field: "cases.available_variation_data",
                value: ["ssm"],
              },
              op: "in",
            },
          ],
`

Reply:
Yes it can 😄